plutus-core/plutus-core.cabal

```diff
@@ -532,6 +532,32 @@ test-suite untyped-plutus-core-test
     , tasty-hunit
     , text
 
+executable untyped-plutus-core-test-dump
```

Comment:
what's this?

Reply:
Oh, thanks for spotting that.  I added an executable to dump UPLC source code from the plutus-core tests (they just construct ASTs directly in Haskell) but accidentally committed the cabal entry for it. I've removed that again now.

Reply:
If we find that helpful maybe we can leave it but we'll need to add README and docs etc.